src/streaming/controllers/BufferController.js

```diff
@@ -131,16 +129,15 @@ function BufferController(config) {
         eventBus.on(Events.PLAYBACK_RATE_CHANGED, onPlaybackRateChanged, this);
         eventBus.on(Events.PLAYBACK_SEEKING, onPlaybackSeeking, this);
         eventBus.on(Events.WALLCLOCK_TIME_UPDATED, onWallclockTimeUpdated, this);
-        eventBus.on(Events.CURRENT_TRACK_CHANGED, onCurrentTrackChanged, this);
+        eventBus.on(Events.CURRENT_TRACK_CHANGED, onCurrentTrackChanged, this, 0);
```

Comment:
It may be nice to have a few preset priorities that the library uses with plenty of space between them. Then third party developers and others in the library can have flexibility to adjust as needed.

Something like:
```
const priorities = {
  low: 0,
  normal: 200,
  high: 500
}
```

Obviously if we have need for more than those we can adjust, but at least that gives people some indication of where we're at.